app/tests/database.py

```diff
@@ -1,8 +1,26 @@
+# WARNING: Order of imports must not be changed!!
+# The block
+# ---
+# from moto import mock_s3
+# s3mock = mock_s3()
+# s3mock.start()
+# ---
+# must remain at the top before any other import,
+# otherwise mocking s3 will not work successfully
+from moto import mock_s3
+s3mock = mock_s3()
+s3mock.start()
+import boto3
+import botocore
```

Comment:
This should be configured in isort to do so

Reply:
How? I added an `isort: ignore` statement for now to be able to `make format`